styles/globals.css

```diff
@@ -1,6 +1,8 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+@import url('https://fonts.googleapis.com/css2?family=Rozha+One&display=swap');
```

Comment:
We might need to confirm the font weights needed for the design system and import them only.